### PR TITLE
Archana_Assignment_21_Done(10min+10min)

### PR DESCRIPTION
**Pull request by ArchanaNJun20**
Program 1: Swap two numbers 10min
image
Program 2: Fibonacci Numbers 10min
image

### CODE REVIEW

File: src/archanaNJun20/FibonacciNumbers.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class FibonacciNumbers {
+	void printFibonacciNumbers(int input){
+		int num1=1;
+		int num2=1;
+		int sum=0;
+		System.out.print("\n"+num1 + "  "+num2+" ");
+		for(int index=2;index<input;index++){
+			sum=num1+num2;
+			num1=num2;
+			num2=sum;
```
Comment (AavrutiDJun20) [inline src/archanaNJun20/FibonacciNumbers.java]: you can calculate Fibonacci series using two variables only.

Comment (ArchanaNJun20) [reply src/archanaNJun20/FibonacciNumbers.java]: The code updated based on the comment. calculated the Fibonacci series without using the third variable.
thanks for your suggestion.please review and approve.

Comment (AavrutiDJun20) [reply src/archanaNJun20/FibonacciNumbers.java]: Good job

**Review by AavrutiDJun20 — APPROVED** [review]

**Comment by KrishnaKTechnocredits** [comment]
Good Job Archana, good comment Avaruti.

**Pull request merged**